course_pages/views.py

```diff
@@ -27,7 +27,7 @@ def courses_index(request, subject=None):
         "universities": annotate_with_public_courses(University.objects.active_by_score()),
         "languages": languages,
         "courses_count_start_soon": Course.objects.start_soon().count(),
-        "courses_count_end_soon": Course.objects.end_soon().count(),
+        "courses_count_end_soon": Course.objects.enrollment_ends_soon().count(),
```

Comment:
Why not replace the template variable `courses_count_end_soon` by `courses_count_enrollment_ends_soon` as well?